### PR TITLE
Chore refactor api cluster tests part 2

### PR DESCRIPTION
**Pull request by dybi**
Converted all tests to support framework, reduced duplication, extracted common parts of the code

### CODE REVIEW

File: concent_api/api-integration-force-payment.py [hunk 1]
```diff
@@ -44,266 +41,224 @@ def subtask_results_accepted(timestamp = None, payment_ts = None, task_to_comput
...
                 )
             ]
         ),
 
-        headers = {
-            'Content-Type':                     'application/octet-stream',
-            'temporary-list-of-transactions':   'True',
-            'temporary-v':                      '-1',
-            'temporary-eth-block':              '-1'
+        headers={
```
Comment (rwrzesien) [inline concent_api/api-integration-force-payment.py]: Would it make sense to make this header default inside `api_request` call ? It is repeated in almost every call.

File: concent_api/api-integration-force-accept-or-reject-results-test.py [hunk 2]
```diff
@@ -2,35 +2,29 @@
...
 
 from api_testing_common import api_request
 from api_testing_common import count_fails
 from api_testing_common import create_client_auth_message
-from api_testing_common import execute_tests
+from api_testing_common import create_signed_task_to_compute
 from api_testing_common import get_task_id_and_subtask_id
-from api_testing_common import get_tests_list
-from api_testing_common import parse_arguments
+from api_testing_common import PROVIDER_PRIVATE_KEY
```
Comment (rwrzesien) [inline concent_api/api-integration-force-accept-or-reject-results-test.py]: What is the idea behind import keys vs generating them ?

File: concent_api/api-integration-force-report-computed-task-test.py [hunk 3]
```diff
@@ -85,46 +61,43 @@ def main():
...
+                REQUESTOR_PRIVATE_KEY,
+                CONCENT_PUBLIC_KEY,
+                ack_report_computed_task(
+                    task_to_compute=task_to_compute
+                ),
+                headers={
+                    'Content-Type': 'application/octet-stream',
+                },
+                expected_status=202,
+                )
```
Comment (Jakub89) [inline concent_api/api-integration-force-report-computed-task-test.py]: Can You please fix styling here.

Comment (dybi) [reply concent_api/api-integration-force-report-computed-task-test.py]: done

**Pull request merged**